citystories.json

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
I don't think this file belongs to this branch

Reply:
Thanks and done